torch/nn/functional.py

```diff
@@ -2050,8 +2050,13 @@ def hardswish(input: Tensor, inplace: bool = False) -> Tensor:
 
 
 def _no_grad_embedding_renorm_(weight: Tensor, input: Tensor, max_norm: float, norm_type: float) -> Tensor:
+    prev_state = get_fwd_grad_enabled()
     with torch.no_grad():
-        torch.embedding_renorm_(weight, input, max_norm, norm_type)
+        set_fwd_grad_enabled(False)
```

Comment:
cc @albanD

Reply:
Sorry for the confusion, Alban and I talked offline and this is a functorch-only function that I'm guessing worked locally through some dynamic linking or something?

This will change in a moment to use detach instead, though it might still be nice to have an equivalent for forward mode